concent_api/core/tests/test_integration_force_payment.py

```diff
@@ -497,59 +498,6 @@ def test_provider_send_force_payment_with_empty_subtask_results_accepted_list_co
         self._test_400_response(response)
         self._assert_stored_message_counter_not_increased()
 
-    def test_provider_send_force_payment_with_empty_requestor_ethereum_public_key_concent_should_refuse(self):
```

Comment:
why was this removed?

Reply:
It is not possible to correctly create `TaskToCompute` with empty `requestor_ethereum_public_key` as its correctness now gets validated during deserialization against `eth_sig`.